R/indicators.R

```diff
@@ -0,0 +1,36 @@
+#' Get indicator matrix from factor
+#'
+#' Takes a factor vector and construct the corresponding indicator matrix, also
+#' known as a one-hot encoding matrix.
+#'
+#' @param x A factor without missing values.
+#'
+#' @return A integer matrix with the same number of columns as levels of `x`,
+#' and the same number of rows as the number of elements in `x`.
+#'
+#' @examples
+#' get_indicators(factor(letters))
+#'
+#' get_indicators(factor(letters[1:2], levels = letters))
+#'
+#' set.seed(1234)
+#' get_indicators(factor(sample(letters[1:4], 10, TRUE)))
+#' @export
+get_indicators <- function(x) {
+  if (!is.factor(x)) {
```

Comment:
Should this work for ordered factors? I don't seen documentation (or checks) either way.

Reply:
Since ordered factors are a subset of factors I would think that they should work.
https://github.com/tidymodels/hardhat/pull/215/commits/76cd0d051beb008df73bc645023bab86d12589e7